pytest_rerunfailures.py

```diff
@@ -186,13 +195,24 @@ def get_reruns_delay(item):
     rerun_marker = _get_marker(item)
 
     if rerun_marker is not None:
+        args = copy(rerun_marker.args).pop(0)
```

Comment:
This being python, I'm trying to follow the types and getting very confused. As far as I can tell, `rerun_marker.args` is a tuple, and this doesn't have a `pop` method. `pop` would also return the first argument, which makes no sense for a variable called `args`.

Reply:
Also, you seem to be treating the `copy` module as a function here which looks wrong (i.e. you did `import copy` and not `from copy import copy`.

Reply:
Sorry I didn't test that bit at all :(

Fixed